homeassistant/components/lifx/util.py

```diff
@@ -80,6 +82,17 @@ def find_hsbk(hass: HomeAssistant, **kwargs: Any) -> list[float | int | None] |
     """
     hue, saturation, brightness, kelvin = [None] * 4
 
+    if (color_name := kwargs.pop(ATTR_COLOR_NAME, None)) is not None:
+        try:
+            hue, saturation = color_util.color_RGB_to_hs(
+                *color_util.color_name_to_rgb(color_name)
+            )
+        except ValueError:
```

Comment:
We should probably let this raise instead since it is invalid input

Reply:
It's not raisaed in `preprocess_turn_on_alternatives` so I didn't want to raise it here either.

Reply:
My goal was to restore the user experience, not change it which raising would do if someone is currently using an invalid color name.